diavlos/src/service/service.py

```diff
@@ -260,8 +263,8 @@ def fetch_by_name(self, name, fetch_bpmn_digital_steps=None):
                     service_dict).replace('\n', '').replace(
                     '\t', '').replace('\"', '\'')
 
-            # data = {**data, **{"update":latest_update_date,"page_id":page_id}}
-            # data = {'data': data, "update":latest_update_date,"page_id":page_id}
+            data = {**data, **{"update":latest_update_date,"page_id":page_id}}
```

Comment:
data variable seems redundant. Just do: `result = {...}`

Reply:
I've refactored this to fix also an issue that Breaks  BPMN #10